envs/example/standard/hosts

```diff
@@ -10,9 +10,16 @@ controller2
 controller1
 controller2
 
+[mongo_db]
+controller1
+controller2
+
 [db_arbiter]
 compute1
 
+[mongo_arbiter]
```

Comment:
It is more flexible to separate them. The db is used for MySQL for other components. Today, it is clear that we have 2 controllers and 1 compute with primary, secondary, and arbiter. When this configuration is changed, potentially the requirements for MySQL and Mongo may or may not be the same. With they are separated, we have one thing less to investigate and worry in the future.